sdk/formrecognizer/Azure.AI.FormRecognizer/src/AnalyzedDocument.cs

```diff
@@ -1,12 +1,24 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
 using Azure.Core;
 
 namespace Azure.AI.FormRecognizer.DocumentAnalysis
 {
     [CodeGenModel("Document")]
     public partial class AnalyzedDocument
     {
+        /// <summary>
+        /// The words that make up this document.
+        /// </summary>
+        public IReadOnlyList<DocumentWord> Words => ClientCommon.GetWords(Pages, Spans);
```

Comment:
I **really** tried using [lazy initialization](https://docs.microsoft.com/en-us/dotnet/api/system.lazy-1?view=net-5.0):
```cs
private Lazy<IReadOnlyList<DocumentWord>>> _words = new(() => ClientCommon.GetWords(Pages, Spans));

public IReadOnlyList<DocumentWord> Words => _words.Value;
```
but I couldn't make it work like this since it won't allow me to call non-static properties during field initialization. We'll eventually need to initialize it in the constructor, which is unfortunate since we'll be overwriting the generated one.

Using `Lazy<T>` also seems ideal since it provides thread-safety.